translation/dest/activity/cs-CZ.xml

```diff
@@ -1,3 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--Generated by crowdin.com-->
-<resources></resources>
\ No newline at end of file
+<resources>
+  <string name="activity">Aktivita</string>
+  <plurals name="solvedNbPuzzles">
+    <item quantity="one">Vyřešen %s taktická úloha</item>
```

Comment:
Proper czech is actually "Vyřešena %s taktická úloha" (notice the trailing "a" at the end of the 1st word).